tools/dds/dds-server/dds-server.cpp

```diff
@@ -105,7 +132,10 @@ void dds_server::post_device_changes(
 
         for( auto dev_to_add : devices_to_add )
         {
-            add_dds_device( dev_to_add.first, dev_to_add.second );
+            if( ! add_dds_device( dev_to_add.first, dev_to_add.second ) )
+            {
+                std::cout << "Error creating a DDS writer" << std::endl;
```

Comment:
LOG_ERROR?

Reply:
Currently we do not use the logger at all, do you want to replace all prints with logs in this PR?

Reply:
No, in the next PR